### ISSUE TITLE
Remote (Github) packages not updated?

### ISSUE THREAD

**Issue by codebygina** [froger-me/wp-packages-update-server#20]
Hi, since the last update of the plugin I'm experiencing this issue and I wonder if it's only me. 

No matter if I delete the package, add it again, it is still stuck to an old version. Github's now version is dummy-plugin 1.50 while the plugin is pulling a mysterious v1.13 :)

Tried to empty cache and everything, still same result. The strange thing is no error is present.

I also tried these steps:
-reset the permalinks
-delete the package from the server
... and checked if the issue still happens when visiting the API url without priming the package with http://mysite.org/wppus-update-api/?action=get_metadata&package_id=dummy-plugin&checking_for_updates=1&update_type=Plugin . Still the same old version pulled.
The version I am stuck with is the one  prior to updating WP Plugin Update Server actually.

Can you, guys, please check if you experience the same on your servers? Thank you!

**Comment by froger-me** [comment]
Hi @wowthemesnet !

I just tested out the Github repo update with the latest version without issue - deleted the existing package, and used the "Prime a package using a remote repository" feature.

At the moment, I don't have time to maintain the plugin - it's a one-man operation during my free time, without professional incentive - please see the note in the README.md:

> Pull requests to update the plugin's Proxy Update Checker Library to ensure compatibility with the latest versions of Plugin Update Checker and WP Update Server Library or to support other PHP versions are now accepted - this plugin is a **personal** project, receiving payment for its maintenance does not fit into my schedule, and I do not plan to spend extensive amount of time maintaining it except for obvious bugs.

If you do find the source and/or a solution for the issue, please feel to fork and send a pull request, or feel free to document it here, as I may be able to patch the code base.

Thanks!

**Comment by codebygina** [comment]
Hi @froger-me , thanks for your quick answer. I perfectly understand the lack of time, I was talking to the community as a whole to see if anyone else experienced this. Luckily I figured it out, so I am answering myself in case anyone runs into the same issue. It wasn't plugin's fault, but mine. **The version pulled was from the repository releases** (!). 

So what I did was removing the releases entirely or if anyone chooses to enable them, make sure you always create a new release so that WPPUS server can see the correct version.

Thanks once again!
